### PR TITLE
Document: PyCharm Config

### PR DESCRIPTION
**Pull request by avisionh**
# Summary

This branch adds PyCharm environment variable configuration instructions. This is to help PyCharm users set-up their environment variables.

# Checklists

### Project-specific checks

> These are additional checks determined necessary according to the analytical quality assurance plan
(see `docs/aqa/aqa_plan.md`).

- [x] The instructions are clear.
- [x] The instructions work.
- [x] There is nothing else required to import environment variables into PyCharm.

### CODE REVIEW

File: .envrc [hunk 1]
```diff
@@ -9,21 +9,21 @@
...
-export PROJECT_DIR=$(pwd)
+source_env ".secrets"
 
 # Add environment variables for the `data` directories
 export DIR_DATA=$(pwd)/data
-export DIR_DATA_EXTERNAL=$DIR_DATA/external
-export DIR_DATA_RAW=$DIR_DATA/raw
-export DIR_DATA_INTERIM=$DIR_DATA/interim
-export DIR_DATA_PROCESSED=$DIR_DATA/processed
+export DIR_DATA_EXTERNAL=$(pwd)/data/external
```
Comment (mammykins) [inline .envrc:23]: why these changes? So that an env variable is not dependent on another?

**Review by mammykins — COMMENTED** [review]
Worked for me

File: .envrc (hunk 1 above)
Comment (avisionh) [reply .envrc:23]: Yup, I think the `.env` file creation treats `$DIR_DATA` literally (or is static) so if we use `$(pwd)` instead, then that should be dynamic according to what your `pwd` is.

**Review by nacnudus — APPROVED** [review]
I, a novice PyCharmer, was able to follow the instructions.  There were some differences in the menu arrangements in my version, but it was no trouble to find my way.  When I restarted PyCharm, it had recognised the environment variables in `.envrc`.
